plugins/cmdmenu.sma

```diff
@@ -351,97 +417,122 @@ public actionCvarMenu(id, key)
 }
 
 displayCvarMenu(id, pos)
 {
 	if (pos < 0)
-		return
+	{
+		return;
+	}
 
-	new menuBody[512]
-	new b = 0
-	new start = pos * 8
+	new menuBody[512];
+	new b = 0;
+	new start = pos * 8;
 
 	if (start >= g_menuSelectNum[id])
-		start = pos = g_menuPosition[id] = 0
+	{
+		start = pos = g_menuPosition[id] = 0;
+	}
 
-	new len = format(menuBody, charsmax(menuBody), g_coloredMenus ? "\yCvars Menu\R%d/%d^n\w^n" : "Cvars Menu %d/%d^n^n", pos + 1, (g_menuSelectNum[id] / 8 + ((g_menuSelectNum[id] % 8) ? 1 : 0)))
+	new len = formatex(menuBody, charsmax(menuBody), g_coloredMenus ? "\yCvars Menu\R%d/%d^n\w^n" : "Cvars Menu %d/%d^n^n", pos + 1, (g_menuSelectNum[id] / 8 + ((g_menuSelectNum[id] % 8) ? 1 : 0)));
 
-	new end = start + 8
-	new keys = MENU_KEY_0
-	new szValue[64]
+	new end = start + 8;
+	new keys = MENU_KEY_0;
+	new szValue[64];
 
 	if (end > g_menuSelectNum[id])
-		end = g_menuSelectNum[id]
+	{
+		end = g_menuSelectNum[id];
+	}
 
 	for (new a = start; a < end; ++a)
 	{
-		get_cvar_string(g_cvarNames[g_menuSelect[id][a]], szValue, charsmax(szValue))
-		keys |= (1<<b)
-		++b
+		get_cvar_string(g_cvarNames[g_menuSelect[id][a]], szValue, charsmax(szValue));
+		keys |= (1<<b);
+		++b;
 		
 		if (g_coloredMenus)
-			len += format(menuBody[len], charsmax(menuBody) - len, "%d. %s\R%s^n\w", b, g_cvarNames[g_menuSelect[id][a]], szValue)
+		{
+			len += formatex(menuBody[len], charsmax(menuBody) - len, "%d. %s\R%s^n\w", b, g_cvarNames[g_menuSelect[id][a]], szValue);
+		}
 		else
-			len += format(menuBody[len], charsmax(menuBody) - len, "%d. %s    %s^n", b, g_cvarNames[g_menuSelect[id][a]], szValue)
+		{
+			len += formatex(menuBody[len], charsmax(menuBody) - len, "%d. %s    %s^n", b, g_cvarNames[g_menuSelect[id][a]], szValue);
+		}
 	}
 
 	if (end != g_menuSelectNum[id])
 	{
-		format(menuBody[len], charsmax(menuBody) - len, "^n9. %L...^n0. %L", id, "MORE", id, pos ? "BACK" : "EXIT")
-		keys |= MENU_KEY_9
+		formatex(menuBody[len], charsmax(menuBody) - len, "^n9. %L...^n0. %L", id, "MORE", id, pos ? "BACK" : "EXIT");
+		keys |= MENU_KEY_9;
 	}
 	else
-		format(menuBody[len], charsmax(menuBody) - len, "^n0. %L", id, pos ? "BACK" : "EXIT")
+	{
+		formatex(menuBody[len], charsmax(menuBody) - len, "^n0. %L", id, pos ? "BACK" : "EXIT");
+	}
 	
-	show_menu(id, keys, menuBody)
+	show_menu(id, keys, menuBody);
 }
 
 public cmdCvarMenu(id, level, cid)
 {
 	if (!cmd_access(id, level, cid, 1))
-		return PLUGIN_HANDLED
+	{
+		return PLUGIN_HANDLED;
+	}
 
-	g_menuSelectNum[id] = 0
+	g_menuSelectNum[id] = 0;
 
 	for (new a = 0; a < g_cvarNum; ++a)
+	{
 		if (access(id, g_cvarMisc[a][0]))
-			g_menuSelect[id][g_menuSelectNum[id]++] = a
+		{
+			g_menuSelect[id][g_menuSelectNum[id]++] = a;
+		}
+	}
 
-	displayCvarMenu(id, g_menuPosition[id] = 0)
+	displayCvarMenu(id, g_menuPosition[id] = 0);
 
-	return PLUGIN_HANDLED
+	return PLUGIN_HANDLED;
 }
 
 loadCvarSettings(szFilename[])
 {
 	if (!file_exists(szFilename)) 
-		return 0
+	{
+		return PLUGIN_CONTINUE;
```

Comment:
It makes no sense to return `PLUGIN_CONTINUE`, it's a private function where the return value is not used (for now), but still returns 0 or 1 because it makes sense to do that. If something should be changed, I would return a bool.